metaserver/server.go

```diff
@@ -17,20 +13,17 @@ import (
 func InitServer(dataDirectory string, logger *log.Logger) http.Handler {
 	router := mux.NewRouter()
 
+	db := mongoDB{}
```

Comment:
Recommendation: Take database as argument to Init(). Possibly scrap the json backend if we never use it.

Reply:
I think we should just scrap the json backend. It's just too much code to maintain effectively while trying to build out other features.

On the database, is there a reason to take it as an argument to init? I could see supporting different backends, but since we're not doing that, I think it should be the server's responsibility to instantiate it.